misc/slack-notification/on-approve.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
because the pod starts for the second time when approving, the log order is reversed:
first: deployment continued
second - deployment approved

pretty minor but this order is backwards and there isn't anything I can do about it in the current situation